### ISSUE TITLE
Use for asp?

### ISSUE THREAD

**Issue by ghost** [SmItH197/SteamAuthentication#247]
is this usable for an asp.net webform website?

**Comment by bman46** [comment]
No, There is already library that already does this and is written for asp.net in C# (https://github.com/TerribleDev/OwinOAuthProviders).

**Comment by ghost** [comment]
> No, There is already library that already does this and is written for asp.net in C# (https://github.com/TerribleDev/OwinOAuthPro

yeah, but thats for registration, i want them to LOG IN, not register

**Comment by bman46** [comment]
I do not think that that is how that works. On the example login page it says login with external account along with login with a local account and the register page mentions nothing of using an external account.

https://github.com/TerribleDev/OwinOAuthProviders/blob/master/OwinOAuthProvidersDemo/Views/Account/Login.cshtml
